regulations/templates/regulations/chrome.html

```diff
@@ -138,12 +138,15 @@ <h2 class="toc-type">Search</h2>
         <div class="drawer-content">
             <form action="{% url 'chrome_search' reg_part %}" method="GET" role="search">
                 <div class="search-box">
-                    <input type="text" name="q" value="{{ q }}" id="search-input" /><button type="submit"><i class="cf-icon cf-icon-search"></i><strong class="icon-text">Search</strong></button>
+                    <input type="text" name="q" value="{{ q }}" id="search-input" title="Search term"/>
+                    <button type="submit">
+                        <i class="cf-icon cf-icon-search"></i><strong class="icon-text">Search</strong>
```

Comment:
I'd heavily lean on towards replacing `i` elements with `span`, especially since the accessibility checkers will think it's trying to be an outdated italic element. But I agree with making another issue  and closing out the PR since everything else looks ok.